### ISSUE TITLE
Cloudfront Invalid Key when requesting track

### ISSUE THREAD

**Issue by explaura** [TooTallNate/node-spotify-web#129]
I tried requesting a track like so:

```
spotify.trackUri(track, function(err, res) {
      if (err) throw err;
      if (!res.uri) return stream.emit('error', new Error('response contained no "uri"'));

      console.log(res.uri);

      spotify.disconnect();
    });
```

...however the track didn't play. I then immediately copied and pasted the generated URI into a browser and received the following:

> Error Code: InvalidKey
> Error Message: Unknown Key

URI:

> http://dsu0uct5x2puz.cloudfront.net/mp3/71b82366197b20b7709681bfb5eed0f800bc1870?Expires=1467194546&Signature=5U1MTSR~m-3JP3cGfxLqYNhU9~IYtWmYLfX0dwQinyDRqYDPb15Y1R1FI--0ofOA-SQyZctGsfXkIjXdYZ3lVmm9DW-7H7WEWek7h16KqAAgMV7vRshSCmAGX-5lNjH-ibaMkcU16M9sybkafV3KtjPJygCT76uvFgVrlvOQwPj_&Key-Pair-Id=APKAIJ6GQLKYCDRS5HFA

The preview clip of this song works fine. Anyone else experiencing this issue/know what's going wrong?

**Comment by jariz** [comment]
Yes, they renamed it to track_uri2, (and seemed to have disabled the S3 bucket for the normal track_uri call), which will then serve you a encoded mp3, which so far hasn't been cracked yet.
I'm afraid you're out of luck.
I could find [some discussion threads of a couple of guys trying to crack it](https://github.com/Shigawire/node-spotify-web/commit/64fbd3620c9cec6238717a3fcf99b97c356ad85e#comments) (but like half of the comments are deleted), so maybe that helps you.
Also, it appears they added a captcha today, making this library (and the above mentioned fork) unusable without modification.
